engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/views/content-view-composite-content-views-list.html

```diff
@@ -67,7 +67,7 @@
           </div>
           <div>
             <a ui-sref="content-views.details.puppet-modules.list({contentViewId: contentViewVersion.content_view.id})" translate>
-              {{ contentViewVersion.puppet_modules.length || 0 }} Puppet Modules
+              {{ contentViewVersion.puppet_module_count || 0 }} Puppet Modules
```

Comment:
These versions on the list page are components see the [nutupane instantiation](https://github.com/katello/katello/blob/8c3b2e6a682a216ff3f25fcf26c3a6eff2e03924/engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/content-view-composite-content-views-list.controller.js#L31-31) and the [factory method](https://github.com/katello/katello/blob/ee0a6af2734ccc960fc8efbc850a492f171d8d0f/engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/content-view.factory.js#L37-37).